client/web/src/enterprise/campaigns/settings/backend.ts

```diff
@@ -96,15 +99,16 @@ export const queryUserCampaignsCodeHosts = ({
             ${campaignsCredentialFieldsFragment}
         `,
         {
+            username: username ?? '', // TODO: why is this required?
             first: first ?? null,
             after: after ?? null,
```

Comment:
.. and these could become:

```suggestion
            username
            first,
            after,
```

then